database/src/migrations/20240202000001_observation_subcount_attribute_tables.ts

```diff
@@ -0,0 +1,193 @@
+import { Knex } from 'knex';
+
+/**
+ * Create new tables:
+ * - observation_subcount
+ * - subcount_critter
+ * - subcount_attribute
+ *
+ * Create new function/trigger:
+ * - tr_observation_subcount_count
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`--sql
+    -------------------------------------------------------------------------
+    -- Create tables
+    -------------------------------------------------------------------------
+    SET search_path = biohub;
+
+    CREATE TABLE observation_subcount(
```

Comment:
might want to make this table name consistent with other sub count tables `subcount_observation`

Reply:
Yeah this was semi-intentional, though I'm not strongly for or against either name.

I was sort of following a cascading naming pattern:
- survey_observation
- observation_subcount
- subcount_attribute
- attribute_type (if this existed)

Reply:
yeah that's fair, if you've got reason for it I'm fine with the table name as is.